cmd/root.go

```diff
@@ -30,6 +30,7 @@ var (
 var port string = "3000"
 var namespace string
 var kubeConfig string = filepath.Join(homedir.HomeDir(), ".kube", "config")
+var castChartVersion string = ""
```

Comment:
empty string is already the zero value for string

```suggestion
var castChartVersion string
```